projects/ccd-case-ui-toolkit/src/lib/shared/components/case-editor/case-edit-submit/case-edit-submit.component.ts

```diff
@@ -184,7 +184,7 @@ export class CaseEditSubmitComponent implements OnInit, OnDestroy {
   }
 
   public showEventNotes(): boolean {
-    return this.eventTrigger.show_event_notes === true;
+    return this.eventTrigger.show_event_notes !== false;
```

Comment:
@DavidJayakumar - is this not the same as `=== true`? If it's a boolean and not equal to `false`, then it must be `true`

Reply:
done